_pages/en/strategy-communications.md

```diff
@@ -59,16 +59,25 @@ The type of stakeholder to target will depend on the type of strategy to endorse
 The above communication work will require to keep a consistent message to stakeholders in order to avoid confusion and maintain credibility of the team's approach.
 
 <!-- markdownlint-disable MD033 -->
+
 | **Activity**  | **Communication methods available** |
 | ------------- | -------------------------------------------- |
-| Approval      | 1. CIO Executive Committee<br> 2. SABR DGO<br> 3. MM Director |
-| Endorsement   | 1. SABR DGO<br> 2. MM Director<br> 3. IITB Governance Committees<br> 4. IITB Forums<br> 5. Peer reviews<br> 6. Briefing Notes |
-| Outreach      | 1. IITB Governance Committees<br> 2. IITB Forums<br> 3. Public Sector Communities<br> 4. Blogs<br> 3. Social Media<br> 4. IITB IITB Newsletter<br> 5. Email |
-| Feedback      | 1. Social Media<br> 2. Blogs<br> 3. IITB Governance Committees<br> 4. IITB Forums<br> 5. Custom Metrics |
+| Approval      | - CIO Executive Committee<br> - SABR DGO<br> - Director |
+| Endorsement   | - SABR DGO<br> - Director<br> - IITB Governance Committees<br> - IITB Forums<br> - Briefing Notes |
```

Comment:
Need to add Peer Reviews as endorsement.
Peer Reviews are formal, we can record comments and dispose of those comments (i.e. record also the dispositions on those comments). It follows the scientific methods when papers get published. It provides with evidence that teams XYZ have been consulted, what they said, and whether the paper/strategy is still in alignment with them or not.

In an organization like ours, senior leaders will want assurances. Those assurances will come from endorsements from specialists or committees that have delegated mandates in reviewing that kind of stuff. This because senior leaders do not have all of the expertise and will rely on others to give it to them (e.g. architecture committee, CoEs,  director YXZ).

I remember DS being very supportive of this kind of peer review in prior department that I've worked with him.

Reply:
I'm okay with putting there but the table is confusing. Should only clearly show who approves, who endorses. Then below in the method, explain the process.

Reply:
Oops, désolé du délais.

D'accord, mais endorsement be sera pas juste des comités ou des forums. Il y aura probablement des équipes (e.g. EA, IT Security, Ops).

Devrait-on alors lister la liste d'équipes (spécialisations) que nous pensions devrons être reconnues?